### PR TITLE
Fix pr 2221

### PR DESCRIPTION
**Pull request by matsko**

### CODE REVIEW

File: modules/angular2/src/router/route_recognizer.ts [hunk 1]
```diff
@@ -57,8 +57,12 @@ export class RouteRecognizer {
     var solutions = ListWrapper.create();
 
     MapWrapper.forEach(this.redirects, (target, path) => {
-      // TODO: "/" redirect case
-      if (StringWrapper.startsWith(url, path)) {
+      // "/" redirect case
+      if (path == '/') {
```
Comment (btford) [inline modules/angular2/src/router/route_recognizer.ts]: Handle the empty string (`""`) case as well.

**Comment by btford** [comment]
Besides the one comment, LGTM – feel free to merge once you've addressed that.

**Comment by btford** [comment]
LGTM, merge

**Comment by matsko** [comment]
Merged as https://github.com/angular/angular/commit/73d152506b6f5e598e3d1e527e5bb4897895fec0

**Comment by btford** [comment]
:clap: Thanks @matsko!

**Pull request closed without merging**